leshan-client-demo/src/main/java/org/eclipse/leshan/client/demo/LeshanClientDemo.java

```diff
@@ -303,6 +315,15 @@ public static void main(final String[] args) {
             }
         }
 
+        // Abort if cu is used without complete X509 config
+        if (cl.hasOption("cu")) {
+            if (!x509config) {
+                System.err.println("cu should be used with ccert and scert for X509 config.");
+                formatter.printHelp(USAGE, options);
+                return;
+            }
```

Comment:
And some checks about `-cu` usage option.

Reply:
OK